frontend/public/storage/components/storage-overview/storage-overview.jsx

```diff
@@ -122,6 +129,25 @@ export class StorageOverview extends React.Component {
     return false;
   }
 
+  _getUtilizationQuery(type, duration) {
+    switch (type) {
+      case 'iops':
+        return UTILIZATION_IOPS_QUERY + duration;
```

Comment:
this could be written prettier.

Reply:
@suomiy Can you please explain, how? Do you mean using objects?

Reply:
either by that or with the switch. And having the duration concatenation only at one place (not 4x times)